src/main/java/org/web3j/codegen/SolidityFunctionWrapperGenerator.java

```diff
@@ -157,12 +121,13 @@ private void generate() throws IOException, ClassNotFoundException {
         List<AbiDefinition> functionDefinitions = loadContractDefinition(absFile);
 
         if (functionDefinitions.isEmpty()) {
-            Console.exitError("Unable to parse input ABI file");
+            exitError("Unable to parse input ABI file");
         } else {
 
             String className = Strings.capitaliseFirstLetter(contractName);
             System.out.printf("Generating " + basePackageName + "." + className + " ... ");
-            new SolidityFunctionWrapper().generateJavaFiles(contractName, binary, abi, destinationDirLocation.toString(), basePackageName);
+            new SolidityFunctionWrapper().generateJavaFiles(contractName, binary, abi,
```

Comment:
please can you start the newline after generateJavaFiles(, instead of mid-way through the parameter list